### DISCUSSION TITLE
Save csv data success but frame fail

### DISCUSSION THREAD

**Discussion by QuinzyW** [openPupil/Open-PupilEXT#11] [Q&A] [answered]
Hi, After choosing the dada and frame folders, only pupil csv data is successfully stored and the frame folder is empty. I have check many times but it still fails. So Why would this happen? 
We use the PupilEXT with OS: win10, CPU: i51400, memory:? 16G.

**Comment by BZandi** [comment, marked as answer]
You cannot log the pupil data (CSV logging -> red button in PupilEXT) while saving the eye images to the disc (blue button in PupilEXT) simultaneously. This feature is not supported as it would bind to many computer resources, especially in stereo-camera mode.

If you need both the eye images and the logged pupil data as CSV, it is a better practice to record in a first step only the eye images (blue button in PupilEXT) but without pupil tracking or CSV recording. Then, disconnect the cameras in PupilEXT by pressing the respective button in the toolbar and import the eye images for offline analysis into PupilEXT. Now you can start logging the pupil diameter into a CSV file (red button in PupilEXT) based on the previously captured eye images that are on your disc. For this, do not forget to load also the camera calibration file, as shown in the video tutorials about offline analysis.

**Comment by QuinzyW** [comment]
Thanks for your help, the question is solved.
